src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -175,10 +175,10 @@ enum WINDOW_CHEATS_WIDGET_IDX {
 #pragma endregion
 
 #define MAIN_CHEATS_WIDGETS \
-    { WWT_FRAME,            0,  0,          WW - 1, 0,      WH - 1,     0xFFFFFFFF,             STR_NONE },                 /* panel / background   */ \
+    { WWT_FRAME,            0,  0,          WW - 1, 0,      WH - 1,     STR_NONE,             STR_NONE },                 /* panel / background   */ \
     { WWT_CAPTION,          0,  1,          WW - 2, 1,      14,         STR_CHEAT_TITLE,        STR_WINDOW_TITLE_TIP },     /* title bar            */ \
     { WWT_CLOSEBOX,         0,  WW - 13,    WW - 3, 2,      13,         STR_CLOSE_X,            STR_CLOSE_WINDOW_TIP },     /* close x button       */ \
-    { WWT_IMGBTN,           1,  0,          WW - 1, 43,     WH - 1,     0xFFFFFFFF,             STR_NONE },                 /* tab content panel    */ \
+    { WWT_IMGBTN,           1,  0,          WW - 1, 43,     WH - 1,     STR_NONE,             STR_NONE },                 /* tab content panel    */ \
```

Comment:
The last argument looks misaligned now (here and in other places).

Reply:
Missed a few occurrences, apparently. Will fix.